crates/cli/src/subcommands/subscribe.rs

```diff
@@ -198,7 +198,7 @@ where
     ws.send(msg.into()).await
 }
 
-/// Await the initial [`ServerMessage::SubscriptionUpdate`].
+/// Await the initial [`ws::ServerMessage::TransactionUpdateLight`] `|` [`ws::ServerMessage::TransactionUpdate`].
```

Comment:
what about the `InitialSubscription` case? It looks it has a `DatabaseUpdate` or something like that?

(Does this function even need a doc comment? I don't think we generate any docs for this)

Reply:
I don't think the new comment is correct. This function is waiting for a `SubscriptionUpdate` message. If it receives a `TransactionUpdate` or `TransactionUpdateLight`, it fails.

```suggestion
/// Await the initial [`ws::ServerMessage::SubscriptionUpdate`].
```

In response to @bfops , doc comments are valuable internally even when we don't actually use the generated docs, as they show up in IDEs while we're developing SpacetimeDB.